src/core/ext/filters/client_channel/lb_policy_factory.c

```diff
@@ -54,10 +54,10 @@ grpc_lb_addresses* grpc_lb_addresses_copy(const grpc_lb_addresses* addresses) {
   return new_addresses;
 }
 
-void grpc_lb_addresses_set_address(grpc_lb_addresses* addresses, size_t index,
-                                   void* address, size_t address_len,
-                                   bool is_balancer, const char* balancer_name,
-                                   void* user_data) {
+void grpc_lb_addresses_set_address(const grpc_lb_addresses* addresses,
```

Comment:
The `addresses` argument should not be `const`, because this method is modifying it.

Reply:
Yes, it is. Though it's modifying in an indirect way thus doesn't trigger any failure, it's more reasonable to remove the `const` to avoid confusion.